### PR TITLE
Thank co-authors in DocsBot messages

### PR DESCRIPTION
**Pull request by delucis**
#### What kind of changes does this PR include?

- Something else!

#### Description

- Don’t cancel the congrats bot if two commits overlap
- Improve the message posted to Discord to thank commit co-authors 🥳

### CODE REVIEW

File: scripts/congrats-bot.mjs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+ */
+function getCoAuthorsMessage(names) {
+	const messages = [
+		'_Thanks <names> for helping!_ ✨',
+		'_<names> stepped up to lend a hand — thank you!_ 🙌',
+		'_<names> with the assist!_ 💪',
+		'_Couldn’t have done this without <names>!_ 💜',
+		'_Made even better by <names>!_ 🚀',
+		'_And the team effort award goes to… <names>!_ 🏆',
+		'_Featuring the one and only <names>!_ 🌟'
```
Comment (sarah11918) [inline scripts/congrats-bot.mjs:57]:
```suggestion
		'_Featuring contributions by <names>!_ 🌟'
```
The one and only... <multiple names>...  ;)

File: scripts/congrats-bot.mjs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+	let coAuthorThanks = '';
+	if (coAuthors.length) {
+		const names = makeList(coAuthors);
+		coAuthorThanks = '\n' + getCoAuthorsMessage(names);
+	}
+
+	setOutput(
+		'DISCORD_MESSAGE',
+		`**Merged!** ${author}: [\`${commitMessage}\`](<https://github.com/withastro/docs/commit/${id}>)${coAuthorThanks}`
```
Comment (aFuzzyBear) [inline scripts/congrats-bot.mjs:31]:
```suggestion
		` 🎉**Merged!**🎉  ${author}: [\`${commitMessage}\`](<https://github.com/withastro/docs/commit/${id}>)${coAuthorThanks}`
```

I think maybe some small emojis around the **Merged** statement would add a nice touch

**Review by aFuzzyBear — COMMENTED** [review]
This workflow looks amazing Chris Great work,

🎉🥳🎉

**Comment by sarah11918** [comment]
LGTM! Just suggested a message that works singular and plural.

**Pull request merged**